### ISSUE TITLE
Tech partnerships with design agencies

### ISSUE THREAD

**Issue by iteles** [dwyl/hq#457]
We have in the past been approached by a few design agencies to form a technical partnership.

For the most part we have said no as we were either in the early stages of forming a relationship with our current design agency partners or largely were not a good fit - the vast majority just want someone to outsource writing their code to and we don't operate in this way. If there is no desire to collaborate and go on a journey together, we're not interested.

We have now been approached by yet another one, but this one with what appears, at the early stages, to be a genuine interest in collaboration.

## Tasks
+ [ ] Initial meeting for introduction
+ [ ] dwyl to send over some example sprint costs, case studies and bullet points on what is important to us in a partnership

Next steps tbd.

@katbow @Cleop @Danwhy @SimonLab @shouston3 @RobStallion @nelsonic You have all worked within our current partnership. It's very different in that our role here was to try to teach a team who had never worked in the agile way before how to do so, but it would still be extremely helpful if you could write a comment with anything that comes to mind that would be a possible concern in another partnership ✨

**Comment by Cleop** [comment]
I think partnerships can work well when **expectations of roles/responsibilities are clear** from the outset. Ie. What is the role of a PO in comparison to a Scrum Master? How much/when should technical people from outside of dwyl be involved in a project if we're being hired as a technical team? When are dwyl allowed/not allowed access to the client's client? Sometimes not having access to the end client can seem prohibitive when we may need to liaise with them on something technical.

Otherwise I've enjoyed this kind of work. It's important to keep communication flowing, deadlines met (as missing them has a knock on effect down the team) and to filter out excessive bureaucracy. If you do I think these relationships can be really fruitful.

**Comment by katbow** [comment]
Agreed with @Cleop, and also expectations from a technical side! e.g setting the expectations for what sized screens will be styled to a T. I think alongside that, setting up some training at the beginning (like dwyl already does for GitHub training) for technical aspects that will be useful in the partnership would be great. 

I think clients the design agency takes on is a factor as well. Even if the design agency is interested in collaborating and working in an Agile way, when the clients are not so interested in this, it's a challenge. 

Overall I think it can make for a great partnership, provided expectations are set from the start on both sides, and both sides continue that communication throughout projects 🙂 .
